py-polars/polars/expr/expr.py

```diff
@@ -8753,6 +8753,102 @@ def ewm_mean(
             self._pyexpr.ewm_mean(alpha, adjust, min_periods, ignore_nulls)
         )
 
+    def ewm_mean_by(
+        self,
+        by: str | IntoExpr,
+        *,
+        half_life: str | timedelta,
+        warn_if_unsorted: bool = True,
+    ) -> Self:
+        r"""
```

Comment:
here's how this looks:

![image](https://github.com/pola-rs/polars/assets/33491632/3e56d4af-ca2c-47d0-9a58-05117756a45d)